scapy/utils6.py

```diff
@@ -780,3 +780,12 @@ def in6_isvalid(address):
         return True
     except:
         return False
+
+def in6_route(addr):
+    """Select IPv6 routes based on the address type"""
+
+    tmp_iface = None
```

Comment:
Maybe this code could go in `conf.route6.route()` directly, without the need for a new function?